orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/instance/TerminatingInstanceSupport.groovy

```diff
@@ -83,9 +87,12 @@ class TerminatingInstanceSupport implements CloudProviderAware {
     String location = stage.context.region
 
     def tsg = oortHelper.getTargetServerGroup(account, serverGroupName, location, cloudProvider)
+    log.info("Target Server Group from oort : {}", objectMapper.writeValueAsString(tsg))
```

Comment:
This seems like an overkill amount of logging to me ... I don't understand what the symptom is that we're trying to dig into?

Reply:
If we need anything, it seems like we only need instance ids? (and perhaps at debug)

Reply:
The issue i see is that when a titus `terminateInstances` task is called , it succeeds but fails to terminate the instance , my guess is this check fails ->(https://github.com/spinnaker/orca/blob/master/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/TerminateInstancesTask.groovy#L61 ) and it just returns a success and i wanted to get the details of what the  `List<TerminatingInstance>` are from `clouddriver` and added these additional logging . Let me know what you think.